src/Sulu/Bundle/WebsiteBundle/Tests/Functional/Sitemap/SitemapXMLGeneratorTest.php

```diff
@@ -55,9 +65,38 @@ public function testSitemap()
         $client->request('GET', 'http://test.lo/sitemap.xml');
         $content = $client->getResponse()->getContent();
 
-        $this->assertContains(
-            '<url><loc>http://test.lo/en</loc><lastmod>2016-03-01</lastmod><xhtml:link rel="alternate" hreflang="en" href="http://test.lo/en"/><xhtml:link rel="alternate" hreflang="x-default" href="http://test.lo/en"/><xhtml:link rel="alternate" hreflang="en-us" href="http://test.lo/en-us"/></url><url><loc>http://test.lo/en-us</loc><lastmod>2016-03-01</lastmod><xhtml:link rel="alternate" hreflang="en" href="http://test.lo/en"/><xhtml:link rel="alternate" hreflang="x-default" href="http://test.lo/en"/><xhtml:link rel="alternate" hreflang="en-us" href="http://test.lo/en-us"/></url>',
+        $this->assertSitemapXml(<<<EOT
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:xhtml="http://www.w3.org/1999/xhtml">
+  <url>
+    <loc>http://test.lo/en</loc>
+    <lastmod>:date</lastmod>
+    <xhtml:link rel="alternate" hreflang="en" href="http://test.lo/en"/>
+    <xhtml:link rel="alternate" hreflang="x-default" href="http://test.lo/en"/>
+    <xhtml:link rel="alternate" hreflang="en-us" href="http://test.lo/en-us"/>
+  </url>
+  <url>
+    <loc>http://test.lo/en-us</loc>
+    <lastmod>:date</lastmod>
+    <xhtml:link rel="alternate" hreflang="en" href="http://test.lo/en"/>
+    <xhtml:link rel="alternate" hreflang="x-default" href="http://test.lo/en"/>
+    <xhtml:link rel="alternate" hreflang="en-us" href="http://test.lo/en-us"/>
+  </url>
+</urlset>
+EOT
```

Comment:
Improved these tests to be more readable and to use the current date /cc @wachterjohannes